### ISSUE TITLE
sith type ambigious

### ISSUE THREAD

**Issue by OR13** [decentralized-identity/keri#34]
`sith = signing threshold or list of weighted thresholds (icp, rot, dip, drt)`

can it be exactly one of these, and not an `or` type?

Prefer it be just `list of weighted thresholds (icp, rot, dip, drt)`... but with some better example data.

**Comment by SmithSamuelM** [comment]
### Mutually Exclusive Multi-Type Support

Two event elements values may be of more than one type. The design decision was primarily practical. Although the idea of multiple types of values for a given element label introduces complexity it was less complexity then adding an additional element with unique label for each type.  

This trade-off happens with the value types are mutually exclusive. This means that should there be two different labels one for each type, then the presence of both labels must be handled. This is either an error or there must be a prioritization of
the labels. If both are present the implementation must decide which one is to be used. Clearly both should never be present so its an error. There is no way to infer the intent of the provider. But how best to enforce the error?  

The code to enforce mutual exclusivity is at least as complex as the code to do a type check switch.  In either case, because both types must separably supported, the code to support both types after the switch or prioritization is the same. But having only one field label with more than one type value means there is no chance of prioritization errors due to ambiguity of intent. One field forces zero ambiguity of intent. In languages like Python that are duck typed, the consensus is that a type switch is preferred to multiple fields.  

In general, an event's serialization should not be the operative representation. This distinction is not a problem for most languages, but does arise frequently in JavaScript where JSON is so closely tied to the operative representation. In non JavaScript implementations extraction from a serialization should handle the type switch and if desirable the representation after extraction may have multiple attribute labels, one for each value type.

In conclusion, the design decision here is that an or-ing of value types in a serialization field element where those value types are mutually exclusive is practically less error prone and less complex than adding mutually exclusive field labels, one for each value type to the serialization.
